src/test/java/ru/ayaz/CommandExecutorTest.java

```diff
@@ -2,95 +2,61 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import ru.ayaz.ru.ayaz.exceptions.InvalidUserCommandException;
 
-import java.util.HashMap;
-import java.util.Map;
-
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.*;
 
 public class CommandExecutorTest {
```

Comment:
&#x1F34E; **Issue**
Well, the logic of your controller is still not separated well enough. You have UserSocketHandler and CommandExecutor classes which basically constitute a controller together. I guess you didn't want to put their logic into a single class because you see two separate responsibilities there:
1. reading and writing to/from socket
2. validating/executing actual user commands

You could probably separate these two responsibilities, but the way they are separated now is not achieving that goal. Right now it looks that UserSocketHandler handles more parts of the responsibility 2 than CommandExecutor itself (and CommandExecutor actually should be the only one taking care of the responsibility 2).
Either combine them into a single class or separate their concerns better.